src/server/jobs/new-order-jobs.ts

```diff
@@ -149,12 +147,11 @@ const newOrderDailyReset = createJob('new-order-daily-reset', '0 0 * * *', async
       ) as failedJudgments,
       SUM(
         -- Make it so we ignore elements before a reset.
-        if (knoir."createdAt" > parseDateTimeBestEffort(u.startAt), 1, 0)
+        if (knoir."createdAt" > parseDateTimeBestEffort(u.startAt) AND knoir."status" NOT IN ('${NewOrderImageRatingStatus.AcolyteCorrect}', '${NewOrderImageRatingStatus.AcolyteFailed}'), 1, 0)
```

Comment:
Not sure why you'd do this. If we wanna calc. full experience, you need to include Acolyte

Reply:
Acolyte ratings don't count towards calculating your fervor, fervor only applies for knights/templar, so in this case we can safely filter `AcolyteCorrect` and `AcolyteFailed` when counting `totalJudgments`